llvm/include/llvm/AsmParser/LLLexer.h

```diff
@@ -17,18 +17,21 @@
 #include "llvm/ADT/APFloat.h"
 #include "llvm/ADT/APSInt.h"
 #include "llvm/Support/SMLoc.h"
+#include "llvm/Support/SourceMgr.h"
 #include <string>
 
 namespace llvm {
   class Type;
   class SMDiagnostic;
-  class SourceMgr;
   class LLVMContext;
 
   class LLLexer {
     const char *CurPtr;
     StringRef CurBuf;
 
+    // The the end (exclusive) of the current token
```

Comment:
```suggestion
    // The end (exclusive) of the current token.
```

Also there is a mismatch here between the comment referring to "current token" and the member name being PrevTok -- which one is it?

Reply:
that slipped through, it is the previous token